integration_test/README.md

```diff
@@ -248,7 +248,7 @@ a hop. The following is sorted roughly in descending order of usefulness.
 ├── logs
 |   ├── build_and_test.txt
```

Comment:
Taking https://btx.cloud.google.com/invocations/0c11e760-fc12-4fa1-b92d-edf225a36034 as an example, it doesn't look like we create the `build_and_test.txt` file anymore (see https://console.cloud.google.com/storage/browser/ops-agents-public-buckets-test-logs/prod/stackdriver_agents/testing/consumer/third_party_apps/test/bullseye_x86_64/2435/20240708-102933/logs). You might want to remove the reference in lines 240–241, too… Maybe refer to `sponge_log.xml` instead?

Reply:
Some of the above is still not done. Added a follow-up comment.